config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_alloydbclusters.alloydb.cnrm.cloud.google.com.yaml

```diff
@@ -505,6 +515,19 @@ spec:
                 - clusterRef
                 - pointInTime
                 type: object
+              secondaryConfig:
+                description: Configuration of the secondary cluster for Cross Region
+                  Replication. This should be set if and only if the cluster is of
+                  type SECONDARY.
+                properties:
+                  primaryClusterName:
```

Comment:
From a usability perspective, this should be a ref I think.  We should be able to configure that in the service mapping.

Reply:
Yes, this is already part of the service mapping in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1279

The presubmit validation expects the third-party terraform-provider-google-beta update in a different PR, hence this new PR